src/utils/mtev_confstr.c

```diff
@@ -122,13 +123,13 @@ int
 mtev_confstr_parse_boolean(const char *input, mtev_boolean *output) {
   if(!strcasecmp(input, "yes") || !strcasecmp(input, "true") || !strcasecmp(input, "on")) {
```

Comment:
Maybe? but I'd put that in a different PR. For what it's worth, those strings were mostly copied from [reconnoiter](https://github.com/circonus-labs/reconnoiter/blob/master/src/modules/httptrap.c#L111).